diff --git a/src/cocoa_window.m b/src/cocoa_window.m
index 2bff22a6..132413f7 100644
--- a/src/cocoa_window.m
+++ b/src/cocoa_window.m
@@ -1899,6 +1899,12 @@ EGLenum _glfwGetEGLPlatformCocoa(EGLint** attribs)
                 type = EGL_PLATFORM_ANGLE_TYPE_METAL_ANGLE;
         }
 
+        if (_glfw.egl.ANGLE_platform_angle_vulkan)
+        {
+            if (_glfw.hints.init.angleType == GLFW_ANGLE_PLATFORM_TYPE_VULKAN)
+                type = EGL_PLATFORM_ANGLE_TYPE_VULKAN_ANGLE;
+        }
+
         if (type)
         {
             *attribs = _glfw_calloc(3, sizeof(EGLint));
