diff --git a/CMakeLists.txt b/CMakeLists.txt
index b2d0276c..4e902c82 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -122,9 +122,14 @@ set(CMAKE_SKIP_INSTALL_ALL_DEPENDENCY true)
 # All projects in this repo should share the same version of 3rd party depends.
 # It's the only way to remain sane.
 set(CMAKE_FILES "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
-set(CMAKE_CXX_STANDARD 17)
 
-set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+if (NOT CMAKE_CXX_STANDARD)
+    set(CMAKE_CXX_STANDARD 17)
+endif()
+
+if (NOT CMAKE_POSITION_INDEPENDENT_CODE)
+    set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+endif()
 
 include(GNUInstallDirs)
 set(LD_TARGETS_EXPORT_NAME ${PROJECT_NAME}Targets)
@@ -170,10 +175,8 @@ endif ()
 
 if (LD_DYNAMIC_LINK_OPENSSL)
     message(STATUS "LaunchDarkly: searching for shared OpenSSL library")
-    set(OPENSSL_USE_STATIC_LIBS OFF)
 else ()
     message(STATUS "LaunchDarkly: searching for static OpenSSL library")
-    set(OPENSSL_USE_STATIC_LIBS ON)
 endif ()
 
 find_package(OpenSSL REQUIRED)
@@ -182,10 +185,8 @@ message(STATUS "LaunchDarkly: using OpenSSL v${OPENSSL_VERSION}")
 
 if (LD_DYNAMIC_LINK_BOOST)
     message(STATUS "LaunchDarkly: searching for shared Boost libraries")
-    set(Boost_USE_STATIC_LIBS OFF)
 else ()
     message(STATUS "LaunchDarkly: searching for static Boost libraries")
-    set(Boost_USE_STATIC_LIBS ON)
 endif ()
 
 if (LD_BUILD_SHARED_LIBS)
@@ -198,10 +199,8 @@ if (LD_BUILD_SHARED_LIBS)
     endif ()
 endif ()
 
-set(Boost_USE_MULTITHREADED ON)
-set(Boost_USE_STATIC_RUNTIME OFF)
-
 find_package(Boost 1.81 REQUIRED COMPONENTS json url coroutine)
+find_package(tl-expected CONFIG REQUIRED)
 message(STATUS "LaunchDarkly: using Boost v${Boost_VERSION}")
 
 set(FOXY_BUILD_TESTING OFF)
@@ -244,7 +243,6 @@ endif ()
 # Built as static or shared depending on LD_BUILD_SHARED_LIBS variable.
 # This target "links" in common, internal, and sse as object libraries.
 add_subdirectory(libs/client-sdk)
-add_subdirectory(libs/server-sdk)
 
 if (LD_BUILD_CONTRACT_TESTS)
     message(STATUS "LaunchDarkly: building contract tests")
diff --git a/vendor/foxy/cmake/certify.cmake b/vendor/foxy/cmake/certify.cmake
index 73545256..dd14b64e 100644
--- a/vendor/foxy/cmake/certify.cmake
+++ b/vendor/foxy/cmake/certify.cmake
@@ -21,7 +21,3 @@ FetchContent_MakeAvailable(certify)
 
 set(BUILD_TESTING ${PREVIOUS_BUILD_TESTING})
 
-install(
-        TARGETS core
-        EXPORT ${LD_TARGETS_EXPORT_NAME}
-)
