--- src/CMakeLists.txt
+++ src/CMakeLists.txt
@@ -31,13 +31,13 @@
 
 # Dependencies
 target_link_libraries(ftdi1 ${LIBUSB_LIBRARIES})
-
+if(NOT STATICLIBS)
 install ( TARGETS ftdi1
           RUNTIME DESTINATION bin
           LIBRARY DESTINATION lib${LIB_SUFFIX}
           ARCHIVE DESTINATION lib${LIB_SUFFIX}
         )
-
+endif()
 if ( STATICLIBS )
   add_library(ftdi1-static STATIC ${c_sources})
   target_link_libraries(ftdi1-static ${LIBUSB_LIBRARIES})
